crypto/nrf_oberon/include/mbedtls/ecjpake_alt.h

```diff
@@ -33,6 +33,7 @@ extern "C" {
 /* @brief Oberon replacement ECJPAKE context */
 typedef struct mbedtls_ecjpake_context {
     mbedtls_ecjpake_role role;                                  //!< Role, either client or server.
+    int point_format;                                           //!< Point format
```

Comment:
This comment does not add value.

Reply:
It will at least be added into the documentation. The rest of the context is by design opaque, but the actual addressable types. Will add changes to the later 3.0.7 version (because building and testing this library again, takes an hour of runtime)